hexa/pipelines/views.py

```diff
@@ -102,9 +105,20 @@ def run_pipeline(
         HttpResponse: Returns a dict with the `run_id` key containing the ID of the created run.
     """
     try:
-        pipeline = Pipeline.objects.get(id=id)
+        # first we try to retrieve by id in case the pipeline webhook_url has not been migrated
```

Comment:
Could you add a #TODO to explain that we have to remove this after the next production release and once our users have been warned.